### DISCUSSION TITLE
Update the Facebook SDK for iOS

### DISCUSSION THREAD

**Discussion by abumalick** [thebergamo/react-native-fbsdk-next#224] [Q&A] [answered]
I just received a mail from facebook about updating Facebook SDK for iOS, and after checking the repository, I didn't find anywhere where we define the needed version of the facebook SDK.

It means that we can use this react native adapter with any version of the sdk? We only need to update the pod lock file?

ref
```
pod 'FBSDKCoreKit', '~> 8.0.0'
pod 'FBSDKLoginKit', '~> 8.0.0'
pod 'FBSDKShareKit', '~> 8.0.0'
pod 'FBSDKGamingServiceKit', '~> 8.0.0'
```

**Comment by mikehardy** [comment, marked as answer]
It's right here: https://github.com/thebergamo/react-native-fbsdk-next/blob/fcd4b2a0730e9a484a670d10d5c15ebdc09e9158/react-native-fbsdk-next.podspec#L16-L29

And an update is in progress but needs review + testing, you can help test by referencing the PR via a github reference to the branch in your package.json then in a postinstall step in package.json doing a `cd node_modules/react-native-fbsdk-next && yarn && yarn build` to get the typescript built.

#221

**Reply by mikehardy** [reply]
Oh, and you most definitely *cannot* use this SDK with any version of the underlying SDKs. they have version numbers for a reason (https://semver.org/) and our CHANGELOG indicates our compatibility. In general you need to use the version this module specifies and *not* reference the SDK directly in your Podfile or build.gradle

**Reply by abumalick** [reply]
Oh, thank you very much, it is really funny that a search in this repository didn't show me this file. Thanks for the quick answer.

I will try it out this weekend inchallah.

**Reply by gf5901** [reply]
Is there a plan to upgrade to v16.0?

**Reply by litinskii** [reply]
@abumalick did you try to update ?

**Reply by abumalick** [reply]
Mmmh, I don’t remember exactly what I did. We upgraded this plugin to 12.1.2 in Dec 2023 so I suppose it bumped native sdk versions. 
Marketing team is not complaining so all good for now.

**Reply by litinskii** [reply]
Thanks. 
Strange, I've updated to version 13 but still have the same issue.

**Reply by qsales2022** [reply]
> Thanks. Strange, I've updated to version 13 but still have the same issue.


have you found any solution ? i am  also having the same problem
